README.md

```diff
@@ -54,6 +54,10 @@ file (assuming the `erlang_ls` executable is available in your `PATH`:
     ;; Enable LSP automatically for Erlang files
     (add-hook 'erlang-mode-hook #'lsp)
 
+    ;; Override the key bindings from erlang-mode to use LSP for completion
+    (eval-after-load 'erlang
+      '(define-key erlang-mode-map (kbd "C-M-i") #'company-lsp))
```

Comment:
Maybe it's worth adding in the comment or somewhere that there is another binding from `company-lsp` to trigger this which is closer to what other IDEs have, which is `C-space`.

Reply:
Uhm, I have `C-space` bound to `set-mark-command`.